rov/camera/cameras.py

```diff
@@ -0,0 +1,49 @@
+import subprocess
+import os
+import time
+
+DEVNULL = open(os.devnull, 'w')
+
+class Cameras(object):
+
+    def __init__(self, resolution='1280x720', framerate=30,
+                 devices=None, port=8080, brightness=16, contrast=32):
+        self.process = None
+        self.resolution = resolution
+        self.framerate = framerate
+        self.port = port
+        self.output = 'output_http.so -p {port} {web}'.format(
+            port=port,
+            web='-w /usr/local/www'
+        )
+        if not devices:
+            devs = list(subprocess.check_output('ls /dev/video*', shell=True).splitlines())
```

Comment:
Why fork and use a shell command? Python has `os.listdir` to do it natively. For example:
`[dev for dev in os.listdir('/dev') if dev.startswith('video')]`

Reply:
It is now using this method.